gateway-ha/src/test/java/io/trino/gateway/ha/TestTrinoResource.java

```diff
@@ -33,6 +32,8 @@
 
 import java.util.Arrays;
 
+import static io.airlift.http.client.HttpStatus.NOT_FOUND;
```

Comment:
Should we use this approach we use here consistently?

Reply:
Are you talking about this? https://github.com/trinodb/trino-gateway/blob/14fc65c2502a2eee584830bc37a98eeae840a7c1/gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStatsHttpMonitor.java#L151-L155

We can't use `NOT_FOUND.code()` in `case`.
> Every case label has a case constant, which is either a constant expression or the name of an enum constant.

Reply:
I just seems to recall that we use a different HttpStatus or hardcoded numbers in some other file .. ideally we are consistent.